src/odemis/util/filename.py

```diff
@@ -102,21 +102,7 @@ def guess_pattern(fn):
                 # That's a real match => change the pattern
                 fn_ptn = fn_ptn[0:m.start()] + name_ptn + fn_ptn[m.end():]
 
-    # Detect count
-    cnt_ptn = r'\d{1,5}'
-    cnt_m = None
-    for m in re.finditer(cnt_ptn, fn_ptn):
-        cnt_m = m
-    # if multiple numbers are present, use last
-    if cnt_m:
-        cnt = cnt_m.group()
-        fn_ptn = fn_ptn[:cnt_m.start()] + "{cnt}" + fn_ptn[cnt_m.end():]
-    else:
-        cnt = "001"  # will be used in case cnt pattern is added afterwards
-
-    # If neither time, nor count are specified, add count, if only short time (h, min)
-    # specified and no count, add seconds to make filename unique
-    # Doesn't behave properly if user enters terms with curly braces like {{cnt}}.
+    cnt = "001"
```

Comment:
If the end number in the filename is a count, then adding a {cnt} separately at the end is not a good method, as you mentioned, "But having "test-123' converted to 'test-123-001' instead of 'test-124' doesn't sound good". In cases where the users have a special name for their files like "bacteria_na_08.tiff", "virus_50x", then in this changing the filename to  "bacteria_na_09.tiff", would not be the best option. I feel the naming by the users can vary across different users. To tackle it, if we do not touch their given name and append the counter at the end, it might be straightforward? For the users,  the order of the acquired images will be clear when the counter is starting from 1 ?

Reply:
Should I leave it like this for time-being till we know the issue or reset my changes?

Reply:
Please drop your changes to `guess_pattern()`, and handle separately the feature number in the `_add_feature_info_filename()` function.